src/map/packet_system.cpp

```diff
@@ -6486,6 +6486,11 @@ void SmallPacket0x0DE(map_session_data_t* const PSession, CCharEntity* const PCh
     _sql->EscapeString(message, PChar->bazaar.message.c_str());
 
     _sql->Query("UPDATE char_stats SET bazaar_message = '%s' WHERE charid = %u", message, PChar->id);
+
+    if (settings::get<bool>("logging.DEBUG_BAZAARS"))
```

Comment:
You added macros for `DebugBazaars`, so why not use them?

```cpp
DebugBazaarsFmt("Bazaar Interaction [Set Message] - Character: {}, Message: '{}'", PChar->name, message);
```

Reply:
> You added macros for `DebugBazaars`, so why not use them?
> 
> ```c++
> DebugBazaarsFmt("Bazaar Interaction [Set Message] - Character: {}, Message: '{}'", PChar->name, message);
> ```

Great question! I was so hung up on making sure my code matched similar code I was using as a reference that I didn't even realize I'd done that. Will update again shortly.

Reply:
> You added macros for `DebugBazaars`, so why not use them?
> 
> ```c++
> DebugBazaarsFmt("Bazaar Interaction [Set Message] - Character: {}, Message: '{}'", PChar->name, message);
> ```

The macros in `logging.h` actually weren't set up quite right. They had an extra `STATEMENT_CLOSE` at the end which wasn't necessary because it was already being done in `LOGGER_ENABLE`. This made the syntax invalid when trying to use them. I went ahead and removed them from all the `*Fmt` macros which hadn't been used anywhere else in the codebase so it seemed safe.